src/CodeViewerDemo/CodeExamples.h

```diff
@@ -23,12 +23,21 @@ inline const QString testing_example =
   kTreeSize == 3;
   _Tree_;
 
+  // Class members
+  Track* track_;
+  __int__;
+  m_Text_y;
```

Comment:
If I am not wrong, the code style here would be `m_PascalCase`. Not sure if that matters.

Reply:
Doesn't matter, but I will change it.